RATapi/controls.py

```diff
@@ -41,7 +40,8 @@ class Controls(BaseModel, validate_assignment=True, extra="forbid"):
     procedure: Procedures = Procedures.Calculate
     parallel: Parallel = Parallel.Single
     calcSldDuringFit: bool = False
-    resampleParams: list[float] = Field([0.9, 50], min_length=2, max_length=2)
+    resampleMinAngle: float = Field(0.9, le=1, gt=0)
+    resampleNPoints: int = Field(50, ge=0)
```

Comment:
I think the matlab is set to be greater than, but not equal to zero. Could you please confirm, and decide what's best. (I appreciate it was translated correctly from what was there).

Reply:
good spot! i think strictly greater than is best